src/workers.json

```diff
@@ -1037,14 +1037,42 @@
             {
               "name": "buffer",
               "type": {
-                "name": "ArrayBufferView"
```

Comment:
Uggh. @mrbbot looks like generics are kind of broken in workers.json generation.